content/en/armory-enterprise/plugin-guide/plugin-external-account.md

```diff
@@ -311,6 +323,10 @@ This plugin can read account credentials in the following layouts:
 
 ## Release Notes
 
+* v0.3.0 (01/30/2022)
```

Comment:
same here, after the plugin release we can update it

Reply:
we already created the release https://github.com/armory-plugins/external-accounts/releases/tag/v0.3.0